.rubocop.yml

```diff
@@ -145,6 +153,9 @@ Layout/MultilineOperationIndentation:
 Style/AccessModifierDeclarations:
   Enabled: false
 
+Style/RedundantReturn:
```

Comment:
Does this suppress the complaint for `return x` at the end of a method? I thought we had decided that was a good convention?

Reply:
It does.  We have enough legacy code where that was a pain to cleanup or disable, and it complains when the return is in an if statement in the middle of the method where the return isn't obvious.  I agree, we should not add more of them, but there were a few situations that I felt warranted the clarity it provides.